src/schema/v2/sorts/show_sorts.ts

```diff
@@ -3,11 +3,10 @@ import { GraphQLEnumType } from "graphql"
 const SHOW_SORTS = {
   END_AT_ASC: { value: "end_at" },
   END_AT_DESC: { value: "-end_at" },
-  FEATURED_ASC: { value: "-featured,-start_at" },
-  FEATURED_DESC: { value: "-featured,-end_at" },
+  FEATURED_ASC: { value: "featured" },
+  FEATURED_DESC: { value: "-featured" },
   NAME_ASC: { value: "name" },
   NAME_DESC: { value: "-name" },
-  PARTNER_ASC: { value: "fully_qualified_name" },
```

Comment:
This is the list of available sorts on gravity.
https://github.com/artsy/gravity/blob/62c498f581ed283ead1a1ba79980e8c759c2d392/app/api/v1/fairs_shows_endpoint.rb#L17

 `fully_qualified_name` doesn't exist there.